### PR TITLE
WOC last 6 cards + WOC Commander decks

### PR DESCRIPTION
**Pull request by Simisays**
Throne of Eldraine cost of the draw ability not working

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/loamcrafter_faun.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Loamcrafter Faun
+ManaCost:2 G
+Types:Creature Satyr Druid
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDiscard | TriggerDescription$ When CARDNAME enters the battlefield, you may discard one or more land cards. When you do, return up to that many target nonland permanent cards from your graveyard to your hand.
+SVar:TrigDiscard:DB$ Discard | DiscardValid$ Land | AnyNumber$ True | Optional$ True | Mode$ TgtChoose | RememberDiscarded$ True | SubAbility$ TrigImmediateTrig
+SVar:TrigImmediateTrig:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ GE1 | RememberObjects$ Remembered | Execute$ TrigReturn | SubAbility$ DBCleanup | TriggerDescription$ When you do, return up to that many target nonland permanent cards from your graveyard to your hand.
+SVar:TrigReturn:DB$ ChangeZone | TgtPrompt$ Select up to that many target nonland permanents from your graveyard | TargetMin$ 0 | TargetMax$ X | Origin$ Graveyard | Hidden$ True | Destination$ Hand | ValidTgts$ Permanent.nonLand+YouCtrl
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/loamcrafter_faun.txt:8]: why would you make it Hidden?

File: forge-gui/res/cardsfolder/upcoming/throne_of_eldraine.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Throne of Eldraine
+ManaCost:5
+Types:Legendary Artifact
+K:ETBReplacement:Other:ChooseColor
+SVar:ChooseColor:DB$ ChooseColor | Defined$ You | AILogic$ MostProminentHumanControls | SpellDescription$ As CARDNAME enters the battlefield, choose a color.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/throne_of_eldraine.txt:5]: wrong logic

File: forge-gui/res/cardsfolder/upcoming/unfinished_business.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Unfinished Business
+ManaCost:3 W W
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TgtPrompt$ Choose target creature card in your graveyard | SubAbility$ DBAttach | ValidTgts$ Creature.YouCtrl | SpellDescription$ Oracle:Return target creature card from your graveyard to the battlefield, then return up to two target Aura and/or Equipment cards from your graveyard to the battlefield attached to that creature. (If the Auras can't enchant that creature, they remain in your graveyard.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/unfinished_business.txt:4]: fix desc

File: forge-gui/res/cardsfolder/upcoming/unfinished_business.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+SVar:DBAttach:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Aura.YouCtrl,Equipment.YouCtrl | AttachedTo$ ParentTarget | TargetMin$ 2 | TargetMax$ 2 | TgtPrompt$ Select two target Equipments or Auras from your graveyard
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/unfinished_business.txt:5]: wrong min

File: forge-gui/res/cardsfolder/upcoming/throne_of_eldraine.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Throne of Eldraine
+ManaCost:5
+Types:Legendary Artifact
+K:ETBReplacement:Other:ChooseColor
+SVar:ChooseColor:DB$ ChooseColor | Defined$ You | AILogic$ MostProminentInComputerDeck | SpellDescription$ As CARDNAME enters the battlefield, choose a color.
+A:AB$ Mana | Cost$ T | Produced$ Chosen | Amount$ 4 | RestrictValid$ Spell.ChosenColor+MonoColor | SpellDescription$ Add four mana of the chosen color. Spend this mana only to cast monocolored spells of that color.
+A:AB$ Draw | Cost$ Mana<3\ChosenColor> T | NumCards$ 2 | SpellDescription$ Draw two cards. Spend only mana of the chosen color to activate this ability.
```
Comment (kevlahnota) [inline forge-gui/res/cardsfolder/upcoming/throne_of_eldraine.txt:7]: @tool4ever don't know where in ComputerUtilMana to filter this like the XColor param, but maybe this is possible?...

> **Cost$ 3 T | ManaPref$ ChosenColor**

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/throne_of_eldraine.txt:7]: Well it's not only a Pref, other colors aren't allowed so I need to do more support

**Review by tool4ever — APPROVED** [review]

**Pull request merged**